migration to new OIDCOP ver 2.x

MdreW opened the pull request:
* updated proxy.conf for new frontend
* updated frontend configurazion
* update run.sh for docker configuration

MdreW commented:
Hi @scolagreco , I have updated run.sh, you can check the docker configuration work fine?

peppelinux commented:
@MdreW I think that's time to drop my forks here
https://github.com/italia/Satosa-Saml2Spid/blob/master/requirements.txt

it seems to me that both satosa and pysaml2 can now be used as they are release in the mainline. this requires tests anyway but spid-sp-test would help in CI for that

I'd also suggest to put the versions of the packages if you agree.

hint: don't use `==` but always `>=` and/or `<=` like `>=2.1.0,<=2.2.0`

MdreW commented:
Ok, @peppelinux , I try to remove your fork and we try to make a single new version :+1:

MdreW commented:
Hi @peppelinux , I have moved Satosa requirement to official package and I have removed pysaml requirements because si part of Satosa requirements.
* Minimum satosa version is set to 8.2.0
* Minimum oidcop version is set to 2.0.0

MdreW commented:
@peppelinux, what do you think? Is time for a new release?
